<?xml version="1.0" encoding="UTF-8"?>
<!-- Vị trí: /ui/widgets/channeldatawidget.ui -->
<!-- Phiên bản: 4.2 (Đổi tên nút Hủy/Làm mới) -->
<ui version="4.0">
 <class>ChannelDataWidget</class>
 <widget class="QWidget" name="ChannelDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="dateLabel">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>dd/MM/yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dayStatusLabel">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Cần tìm</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Lưu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Mặc định</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Hoàn tác</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="middleLayout">
     <item>
      <widget class="QLineEdit" name="urlLineEdit">
       <property name="placeholderText">
        <string>Link Video...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="titleLineEdit">
       <property name="placeholderText">
        <string>Tiêu đề...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClearableLineEdit" name="descriptionLineEdit">
       <property name="placeholderText">
        <string>Mô tả (nhấn đúp để sửa)...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClearableLineEdit" name="categoryLineEdit">
       <property name="placeholderText">
        <string>Danh mục...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClearableLineEdit" name="playlistLineEdit">
       <property name="placeholderText">
        <string>Playlist...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <widget class="QLabel" name="label_tags">
       <property name="text">
        <string>Tags:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="tagsLayout"/>
     </item>
     <item>
      <widget class="QLabel" name="label_subtags">
       <property name="text">
        <string>Tags con:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="subTagsLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/clearablelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>

